### PR TITLE
Issue 9/show player pvp and pve stats

### PR DESCRIPTION
**Pull request by angelika-c**

### CODE REVIEW

File: src/main/java/com/github/fairit/destinypro/controller/DatabaseController.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package com.github.fairit.destinypro.controller;
 
-import com.github.fairit.destinypro.service.definitions_service.ClassDefinitionService;
-import com.github.fairit.destinypro.service.definitions_service.GenderDefinitionService;
-import com.github.fairit.destinypro.service.definitions_service.RaceDefinitionService;
+import com.github.fairit.destinypro.serviceimpl.definition.ClassDefinitionServiceImpl;
```
Comment (pbielicki) [inline src/main/java/com/github/fairit/destinypro/controller/DatabaseController.java:3]: @angelika-c  `service.impl.definition`

File: src/main/java/com/github/fairit/destinypro/controller/RestApiExceptionHandler.java [hunk 2]
```diff
@@ -1,8 +1,7 @@
 package com.github.fairit.destinypro.controller;
 
-import com.github.fairit.destinypro.dto.exception.ApiErrorDto;
-import com.github.fairit.destinypro.exception.ApiNotFoundException;
-import com.github.fairit.destinypro.exception.CharacterNotFoundException;
+import com.github.fairit.destinypro.dto.exception.ErrorDto;
+import com.github.fairit.destinypro.exception.*;
```
Comment (pbielicki) [inline src/main/java/com/github/fairit/destinypro/controller/RestApiExceptionHandler.java:4]: avoid `.*` imports

File: src/main/java/com/github/fairit/destinypro/dto/playerapi/Response.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.github.fairit.destinypro.dto.player.api;
+package com.github.fairit.destinypro.dto.playerapi;
```
Comment (pbielicki) [inline src/main/java/com/github/fairit/destinypro/dto/playerapi/Response.java:1]: inconsistent with other changes - I'd change this back to `player.api`

File: src/test/java/com/github/fairit/destinypro/serviceimpl/character/CharacterApiServiceTest.java [hunk 4]
```diff
@@ -18,10 +18,10 @@
 public class CharacterApiServiceTest {
 
     @Autowired
-    private CharacterApiService characterApiService;
+    private CharacterApiServiceImpl characterApiService;
```
Comment (pbielicki) [inline src/test/java/com/github/fairit/destinypro/serviceimpl/character/CharacterApiServiceTest.java:21]: use interface only i.e. `CharacterApiService`

change in all other places

File: src/main/java/com/github/fairit/destinypro/controller/DatabaseController.java (hunk 1 above)
Comment (angelika-c) [reply src/main/java/com/github/fairit/destinypro/controller/DatabaseController.java]: done

File: src/main/java/com/github/fairit/destinypro/controller/RestApiExceptionHandler.java (hunk 2 above)
Comment (angelika-c) [reply src/main/java/com/github/fairit/destinypro/controller/RestApiExceptionHandler.java:4]: done

File: src/test/java/com/github/fairit/destinypro/serviceimpl/character/CharacterApiServiceTest.java (hunk 4 above)
Comment (angelika-c) [reply src/test/java/com/github/fairit/destinypro/serviceimpl/character/CharacterApiServiceTest.java]: done

File: src/main/java/com/github/fairit/destinypro/dto/playerapi/Response.java (hunk 3 above)
Comment (angelika-c) [reply src/main/java/com/github/fairit/destinypro/dto/playerapi/Response.java]: done

**Pull request merged**